### PR TITLE
Implement refactoring tag

### PR DESCRIPTION
**Pull request by jasonvank**
command:
**refactorTag t/OLD_TAG [t/NEW_TAG]**
**rt t/OLD_TAG [t/NEW_TAG]**

>e.g. 
refactorTag t/python nt/java (replace `python` tag by `java` tag)
refactorTag t/python (remove python tag in all entries)
rt t/python nt/java (command alias)

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -2,7 +2,6 @@
 ifdef::env-github,env-browser[:relfileprefix: docs/]
 
 https://travis-ci.org/CS2103-AY1819S1-T12-4/main[image:https://travis-ci.org/CS2103-AY1819S1-T12-4/main.svg?branch=master[Build Status]]
-https://coveralls.io/github/CS2103-AY1819S1-T12-4/main?branch=master[image:https://coveralls.io/repos/github/CS2103-AY1819S1-T12-4/main/badge.svg?branch=master[Coverage Status]]
```
Comment (zhengwei143) [inline README.adoc]: Not sure if we should be removing this?

Comment (zhengwei143) [reply README.adoc]: actually dont see the badge on our readme ><

Comment (jasonvank) [reply README.adoc]: I will add back.

Comment (jasonvank) [reply README.adoc]: I add this badge yesterday, it fail my PR so I close it. Will open it again.

**Pull request merged**